spec.bs

```diff
@@ -761,25 +761,40 @@ following [=struct/items=]:
   :: a [=boolean=], initially false
 </dl>
 
+An <dfn for=fencedframetype>automatic beacon event type</dfn> is either "<dfn
+for="automatic beacon event type">`reserved.top_navigation_start`</dfn>", "<dfn
+for="automatic beacon event type">`reserved.top_navigation_commit`</dfn>", or "<dfn
+for="automatic beacon event type">`reserved.top_navigation`</dfn>".
+
+Advisement: <code>[=automatic beacon event type/reserved.top_navigation=]</code> is an earlier
+naming of <code>[=automatic beacon event type/reserved.top_navigation_commit=]</code>. While they
+both do the same thing, <code>[=automatic beacon event type/reserved.top_navigation=]</code> will be
+depreacted in the future and should not be used for new code.
```

Comment:
Two things here.

1. Deprecated is spelled wrong
2. It is deprecated *now*, so let's say it will be **removed** in the future (i.e., we'll follow-up on that deprecation).